package.json

```diff
@@ -48,6 +48,7 @@
     "react-bootstrap": "^0.27.2",
     "react-dom": "^0.14.3",
     "react-hot-loader": "^1.3.0",
+    "react-onclickoutside": "^4.5.0",
```

Comment:
I don't know. I thought since it's small enough, we don't need to increase the number of runtime dependencies and could just bundle it. I can change it if you think it makes a difference.